helx/charts/appstore/templates/configmap.yaml

```diff
@@ -30,4 +30,5 @@ data:
   TEST_USERS_PATH: "{{ .Values.django.TEST_USERS_PATH }}"
   REACT_APP_SEMANTIC_SEARCH_ENABLED: "{{ .Values.helx_ui.REACT_APP_SEMANTIC_SEARCH_ENABLED }}"
   REACT_APP_WORKSPACES_ENABLED: "{{ .Values.helx_ui.REACT_APP_WORKSPACES_ENABLED }}"
-  REACT_APP_HELX_SEARCH_URL: "{{ .Values.helx_ui.REACT_APP_HELX_SEARCH_URL }}"
+  REACT_APP_HELX_SEARCH_URL: '{{ .Values.helx_ui.REACT_APP_HELX_SEARCH_URL }}'
+  REACT_APP_UI_BRAND_NAME: {{ .Values.helx_ui.REACT_APP_UI_BRAND_NAME }}
```

Comment:
@cschreep - Got an error while trying this. Needs quotes around this.

Reply:
That's generally the kind of thing it's good to check before approving the PR :)